gnucash/report/standard-reports/transaction.scm

```diff
@@ -870,17 +870,24 @@ tags within description, notes or memo. ")
       gnc:pagename-display (N_ "Sign Reverses")
       "m1" (_ "Reverse amount display for certain account types.")
       'global
-      (keylist->vectorlist sign-reverse-list))))
+      (keylist->vectorlist sign-reverse-list)))
+
+    (gnc:register-trep-option
+     (gnc:make-internal-option "__trep" "unique-transactions" #f)))
 
   (gnc:options-set-default-section options gnc:pagename-general)
   options)
 
 ;; ;;;;;;;;;;;;;;;;;;;;
 ;; Here comes the big function that builds the whole table.
 
-(define (make-split-table splits options)
+(define (make-split-table splits options custom-calculated-cells)
 
-  (define (opt-val section name) (gnc:option-value (gnc:lookup-option options section name)))
+  (define (opt-val section name)
+    (let ((option (gnc:lookup-option options section name)))
+      (if option
+          (gnc:option-value option)
+          (error (format #f "cannot find ~a ~a" section name)))))
```

Comment:
I'll note the use of error again so I've looked it up in the guile manual to be sure.

First off, it looks like the format specifier is redundant ? The error function takes a message and arguments as parameters.
Secondly, it's not entirely clear to me what this would do and where the error message would get written. Do you want this message to be captured in the gnucash trace file ? Do you want it displayed in the html to the user ?

(I can't easily pull your PR from here, my laptop is not properly configured for that, so I can only ask...).

Reply:
Hmm from memory they were displayed with --debug. I'll modify to gnc:error.